### ISSUE TITLE
Deploy keyboard navigable version of this sim

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/molecules-and-light#162]
We are ready to proceed with a minor release off of master with keyboard navigation for this sim. With the recent chipper 2.0 work, we grabbed a stable SHA of chipper to use with new RC's. Here it is:
```
  "chipper": {
    "sha": "6f8b4c139ef54b626df1ac052874826d4b646cc0",
    "branch": "master"
  },
```
@zepumph volunteered to prepare this RC, many thanks and assigning to him.

**Comment by jessegreenberg** [comment]
@phet-steele just showed me a multi touch issue while interacting with both a keyboard and a mouse, deployment should be on-hold until those are generally resolved.

**Comment by jessegreenberg** [comment]
Blocking issue: https://github.com/phetsims/scenery/issues/713

**Comment by jessegreenberg** [comment]
We also want to pull in #163.

**Comment by jessegreenberg** [comment]
While we still want to bring in #163, phetsims/scenery#713 is sufficiently fixed to no longer be a blocking issue.

**Comment by zepumph** [comment]
On hold until we are done with friction, issues have come up that will effect this too, so I will take shas from master after friction has been published.

**Comment by zepumph** [comment]
https://github.com/phetsims/friction/issues/47 should be close to closing.

**Comment by jessegreenberg** [comment]
Friction has been deployed, this sim is no longer on hold. We need to clean up "group" focus highlights, and then this sim should be ready for a release candidate.

**Comment by jessegreenberg** [comment]
On request that came up was to make sure the spacing was the same between focus highlights and outer highlights for the radio buttons.

![no-space](https://user-images.githubusercontent.com/6396244/34268418-d8515172-e63d-11e7-84f9-d2925a78ff34.PNG)
![more-space](https://user-images.githubusercontent.com/6396244/34268420-d9572d62-e63d-11e7-956d-8288255ce4e0.PNG)

This is happening because the molecule radio buttons have a *larger* highlight so that the focused buttons look OK with the rounded panel corners.
![capture](https://user-images.githubusercontent.com/6396244/34269856-3e0e28aa-e643-11e7-92d8-7e8cbc60016e.PNG)

Could be handled in a number of ways, but IMO best way is to make the outer highlight larger in this case. So we will need to support modifying the outer highlight like we can with our regular focus highlights.

**Comment by jessegreenberg** [comment]
This was done in the  above commits. @terracoda @emily-phet @arouinfar @amanda-phet @zepumph can someone please spot  check the look of the outer highlights in https://www.colorado.edu/physics/phet/dev/html/molecules-and-light/1.4.0-dev.5/molecules-and-light_en.html?accessibility

Are we ready for an RC?

**Comment by arouinfar** [comment]
@jessegreenberg the spacing looks great, but the outer highlight looks quite a bit different than Rutherford Scattering (lighter color and thicker stroke weight). I've missed one or two recent keyboard nav meetings, so it's possible I missed the relevant discussion, but I wasn't sure if the difference was intentional or not. Regardless, I think it's looking pretty nice!

**Comment by jessegreenberg** [comment]
Discussed on 1/2/18:

We like the current look in MAL, but wanted to change the way the outer highlight works so that it looks good in sims with dark backgroudns (like rutherford-scattering). We are going to remove the transparency of the outer highlight, and have two colors, one for dark backgrounds and one for light.

**Comment by jessegreenberg** [comment]
Here are the colors that we decided on during some in-meeting prototyping:
For light backgrounds:
`OUTER_GROUP_FOCUS_COLOR = new Color( 'rgba(233,113,166,1.0)' );`
For dark backgrounds:
`OUTER_GROUP_FOCUS_COLOR = new Color( 'rgb(159, 15, 80, 1.0)' );`

**Comment by jessegreenberg** [comment]
This was done in the above commit, which also includes support  for changing all focus highlight colors dynamically. This is required for sims with color profiles (like rutherford-scattering), where we need the darker group highlight color over black backgrounds and the lighter one over other white background.

I think we are ready for an RC for keyboard nav. But I am not sure about the state of build tools, ill ask on slack.

**Comment by zepumph** [comment]
Unassigning, let me know if you need me.

**Comment by jessegreenberg** [comment]
RC prepared here: https://github.com/phetsims/QA/issues/84

**Comment by jessegreenberg** [comment]
RC finished QA testing, and the one issue that was found has a workaround. We are ready for the next RC.

**Comment by jessegreenberg** [comment]
RC spot check deployed in https://github.com/phetsims/QA/issues/89. On hold until that issue is closed.

**Comment by jessegreenberg** [comment]
A new RC spot check deployed in phetsims/QA#90. On hold until that issue is closed.

**Comment by jessegreenberg** [comment]
Simulation was deployed. This issue can be closed.
